### ISSUE TITLE
Theme for linux terminals

### ISSUE THREAD

**Issue by kioplato** [liuchengxu/space-vim-dark#10]
Hello,
is it possible to create a theme for Linux terminals?
The .terminal file, I believe, only works for MacOSX.
Is there a workaround for Linux users? I personally use gnome-terminal.

Thank you.

**Comment by liuchengxu** [comment]
Yes, I'll see what I can do when I get a chance. Recently I use macOS heavily :).

**Comment by liuchengxu** [comment]
Based on [one-gnome-terminal](https://github.com/denysdovhan/one-gnome-terminal), I have made an experimental one for gnome terminal : [gnome](https://github.com/liuchengxu/space-vim-dark/tree/gnome). After running `space-vim-dark.sh` and restart the terminal, you should be able to see the effect. Let me know if you have any ideas.

**Comment by axvr** [comment]
I will be happy to create a linux theme for X based terminals and applications (e.g. URxvt, Xterm, and more). I have already created a starter version which I use, so it should be no problem.

**Comment by liuchengxu** [comment]
@axvr Great! It would be nice to have these support.
